app/javascript/controllers/blogs_controller.js

```diff
@@ -0,0 +1,40 @@
+// Reconnect ActionCable after switching accounts
+
+import { Controller } from "@hotwired/stimulus"
+import consumer from "../channels/consumer"
+
+export default class extends Controller {
+  // connect() {
+  // }
+
+  toggleSwitchListener(event) {
+    const csrfToken = document.querySelector('meta[name="csrf-token"]').getAttribute('content');
```

Comment:
Use `this.csrfToken` in `connect` callback. It will be helpful for catering other fetch API requests without making constants everytime inside of callbacks

Reply:
> Use `this.csrfToken` in `connect` callback. It will be helpful for catering other fetch API requests without making constants everytime inside of callbacks

Fixed this.